docs/source/api/aihwkit.simulator.rpu_base.rst

```diff
@@ -1,16 +0,0 @@
-aihwkit.simulator.rpu\_base module
-==================================
-
-.. automodule:: aihwkit.simulator.rpu_base
-   :members:
-   :undoc-members:
-   :show-inheritance:
-
```

Comment:
Is this on purpose deleted?

Reply:
This is automatically generated using make apidoc.   I can take a look to see why it was removed.